Project.toml

```diff
@@ -8,8 +8,8 @@ Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 
 [compat]
-BinaryProvider = "≥ 0.4.0"
-julia = "≥ 1.0.0"
+BinaryProvider = "0.4, 0.5"
```

Comment:
Allow also v0.6:
```suggestion
BinaryProvider = "0.4, 0.5, 0.6"
```

Reply:
0.6 is a breaking release, so shouldn't we wait till we verify the compatibility? It should be easy to update and incude 0.6 when the release is tagged